src/components/Product.jsx

```diff
@@ -33,136 +37,236 @@ const ProductDetails = () => {
-											alt="images"
-										/>
-									</div>
-									<div
-										className="image-card"
-										onClick={() =>
-											handleSelectedImage(
-												product.images[3],
-											)
-										}
-									>
-										<img
-											src={product.images[3]}
-											alt="images"
-										/>
-									</div>
-									<div
-										className="image-card"
-										onClick={() =>
-											handleSelectedImage(
-												product.images[0],
-											)
-										}
-									>
-										<img
-											src={product.images[0]}
-											alt="images"
-										/>
-									</div>
-								</>
-							)}
-						</div>
-					</div>
-					<div className="description-container">
-						<div className="product-title-price">
-							<h1>
-								{product.name} - Max / ${product.price}
-							</h1>
-						</div>
-						<div className="product-description">
-							<p>
-								combine high-fidelity audio with
-								industry-leading Active Noise Cancellation to
-								deliver an unparalleled listening experience
-							</p>
-							<img src={ratings} alt="images" />
-						</div>
-						<div className="choose-color">
-							<p>Choose a color</p>
-							<div className="color-to-choose">
-								<img src={Circle1} alt="images" />
-								<img src={Circle2} alt="images" />
-								<img src={Circle3} alt="images" />
-								<img src={Circle4} alt="images" />
-								<img src={Circle} alt="images" />
+				<>
+					<div className="container">
+						<div className="image-container">
+							<div className="selected-image-cards">
+								<img src={selectedImage} alt="image" />
+							</div>
+							<div className="image-to-choose">
+								{product.images && product.images.length > 0 ? (
+									<>
+										<div
+											className="image-card"
+											onClick={() =>
+												handleSelectedImage(
+													product.images[1],
+												)
+											}
+										>
+											<img
+												src={product.images[1]}
+												alt="images"
+											/>
+										</div>
+										<div
+											className="image-card"
+											onClick={() =>
+												handleSelectedImage(
+													product.images[2],
+												)
+											}
+										>
+											<img
+												src={product.images[2]}
+												alt="images"
+											/>
+										</div>
+										<div
+											className="image-card"
+											onClick={() =>
+												handleSelectedImage(
+													product.images[3],
+												)
+											}
+										>
+											<img
+												src={product.images[3]}
+												alt="images"
+											/>
+										</div>
+										<div
+											className="image-card"
+											onClick={() =>
+												handleSelectedImage(
+													product.images[0],
+												)
+											}
+										>
+											<img
+												src={product.images[0]}
+												alt="images"
+											/>
+										</div>
+									</>
```

Comment:
Starting from line 73 till here can be wrapped in a array.map functionality hence reducing lines of codes